pkg/storaged/clevis-luks-passphrase.sh

```diff
@@ -30,10 +40,14 @@ if cryptsetup isLuks --type luks1 "$DEV"; then
 elif cryptsetup isLuks --type luks2 "$DEV"; then
     for id in `cryptsetup luksDump "$DEV" | sed -rn 's|^\s+([0-9]+): clevis|\1|p'`; do
         tok=`cryptsetup token export --token-id "$id" "$DEV"`
-        jwe=`jose fmt -j- -Og jwe -o- <<<"$tok" | jose jwe fmt -i- -c`
+        jwe=`printf '%s\n' "$tok" | jose fmt -j- -Og jwe -o- | jose jwe fmt -i- -c`
 
         if pt=`echo -n "$jwe" | clevis decrypt`; then
-            echo $pt
+            if [ "$type" = yes ]; then
+                echo clevis
```

Comment:
Same here, `$pt` is ignored. It feels like the whole script could short-circuit all the program calls and directly exit with "echo clevis" if `--type` is given. Which then bears the question how much of the script actually remains to justify calling it in the first place in the `--type` case?

Reply:
See above. This script should only output "clevis" if there is at least one clevis slot that actually works right now. This is so that the UI can adjust nicely before trying to run `clevis luks unlock`.

(Hmm, `clevis luks unlock` has a -t option for testing a slot... we could use that in a loop over all slots. But, hmm, hmm, that uses cryptsetup --test-passphrase, which is suuuper slow...)

Reply:
So, again, I wouldn't make any further changes.